assets/state-driver/0300_rolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: nvidia-driver
+  namespace: openshift-sro
```

Comment:
the namespaces need to specify openshift, right?

Reply:
Yes, no need for the openshift namespace.